workerbee/base.py

```diff
@@ -1,6 +1,56 @@
-from functools import partial
+import datetime
+import logging
 import random
+import time
 import sys
+from functools import partial
+
+
+DEFAULT_LOGGER = logging.getLogger('workerbee_default')
```

Comment:
'workerbee_default' looks a little grim in stdout:
<img width="1116" alt="screenshot 2016-08-20 15 04 09" src="https://cloud.githubusercontent.com/assets/1312873/17831683/6be1c7a0-66e7-11e6-9dfe-900a837253af.png">
Is the `_default` prefix a standard logging pattern? If not maybe `workerbee`?